clang/include/clang/AST/Decl.h

```diff
@@ -2013,6 +2013,12 @@ class FunctionDecl : public DeclaratorDecl,
     DefaultedFunctionInfo *DefaultedInfo;
   };
 
+  /// Message that indicates why this function was deleted.
+  ///
+  /// FIXME: Figure out where to actually put this; maybe in the
+  /// 'DefaultedInfo' above?
+  StringLiteral *DeletedMessage;
```

Comment:
Yes, this is my main concern with this PR, it does seem pretty wasteful.
DefaultedFunctionInfo seem like a good approach to explore

Reply:
Yeah, this here is definitely just a temporary ‘solution’ to get this working well enough to where it’s testable. We definitely don’t want to leave this here...

Reply:
Just refactored that; see my comment below.